apps/client/src/pages/public/error.tsx

```
@@ -0,0 +1,64 @@
+};
+
+export const ErrorPage = () => {
+    const error = useRouteError() as RouterError;
+
+    return (
+        <LocaleProvider>
+            <main className="flex items-center justify-center min-h-screen p-4">
+                <div className="w-full max-w-sm space-y-6">
+                    <h4 className="flex flex-col text-4xl font-bold text-white">
+                        <span>
+                            {t`Error ${error.status}`}
```

ESLint

[error] Should be ${variable}, not ${object.property} or ${myFunction()}

(lingui/no-expression-in-message)